dashboards-observability/public/components/visualizations/charts/maps/treemaps.tsx

```diff
@@ -87,10 +87,10 @@ export const TreeMap = ({ visualizations, layout, config }: any) => {
     return <EmptyPlaceholder icon={visMetaData?.icontype} />;
```

Comment:
Could we destruct dataConfig like bar chart did and go through all visualizations to change to bar's implementations?

Reply:
We have created another issue for this improvement #1058 we will create another PR for this since this will increase number of changes in this PR.